azure-iot-device/tests/common/pipeline/helpers.py

```diff
@@ -78,7 +78,10 @@ def stage_execute_op(self, op):
         else:
             assert False
 
-    first_stage = stage_to_make()
+    if stage_to_make == pipeline_stages_base.PipelineRootStage:
+        first_stage = stage_to_make(None)
```

Comment:
>first_stage = stage_to_make(None) [](start = 8, length = 33)

Can you mark this line with `# BKTODO: make this more generic` please?  I have a better fix for this and I don't want to lose the spot.

Reply:
And I thought this hack was in 2 places but I can't find the other one now.  If there is another place, can you mark it also please?

---
In reply to: [335153070](https://github.com/Azure/azure-iot-sdk-python/pull/331#discussion_r335153070) [](ancestors = 335153070)

Reply:
found the other points in the workaround and added this.